pom.xml

```diff
@@ -300,6 +299,16 @@
         <artifactId>annotation-indexer</artifactId>
         <version>1.12</version>
       </dependency>
+      <dependency>
+        <groupId>com.fasterxml.jackson.datatype</groupId>
+        <artifactId>jackson-datatype-json-org</artifactId>
+        <version>2.8.11</version>
```

Comment:
Depending on this separately from jackson2-api could cause problems if the API plugin is updated and is no longer compatible with 2.8.11, but I don't think we have any great option since bundling that lib in jackson2-api would add an `org.apache.geronimo.bundles:json` dependency to all users, which also seems undesirable.

Reply:
Yeah, pretty much my thinking.